client/components/select-dropdown/index.jsx

```diff
@@ -207,12 +207,20 @@ class SelectDropdown extends Component {
 			} );
 		}
 
-		let dropdownClassName = classNames( dropdownClasses );
-		let selectedText = this.props.selectedText
-			? this.props.selectedText
-			: result( find(
-				this.props.options, { value: this.state.selected }
+		const dropdownClassName = classNames( dropdownClasses );
+
+		let selectedText = this.props.selectedText;
```

Comment:
I've update the pull request.

Is it ok that now other commits are bundled up in this pull request?

I've not great experience with rebase in Git, and I'm not sure I've followed the process correctly. @gwwar Could you please review the steps, and correct me if I did something wrong/not necessary?

I followed the following steps:

On my **master** branch

```
git pull Automattic master
git push
```

On the **fix/issue-8913** branch

```
git rebase master
```

At this point the status was somehow not consistent:

```
#  Your branch and 'origin/fix/issue-8913' have diverged,
#  and have 10 and 1 different commits each, respectively.
```

I thought i've to do a rebase also with origin/fix/issue-8913

```
git rebase origin/fix/issue-8913
```

This has restored the state, and I was able to proceed; I edited the file, commit, squash, and push.

ps. I don't care too much about the attribution of the commit (I'm here primarily for learning), if I screwed up the pull request seriously feel free to discard it, and restore the fix in a different branch.